MLS.Agent.Tests/TelemetryTests.cs

```diff
@@ -73,21 +73,26 @@ public void Dispose()
         }
 
         [Fact]
-        public async Task TelemetryCommandIsValid()
+        public async Task Jupyter_standalone_command_is_valid()
```

Comment:
"is valid" is pretty vague. The test is actually verifying that the command, when invoked without arguments, sends telemetry.

Reply:
Yea, it is vague. I should be more descriptive, "is_valid" => "sends_telemetry".